debian/control

```diff
@@ -0,0 +1,152 @@
+Source: acrn-hypervisor
+Priority: optional
+Maintainer: Helmut Buchsbaum <helmut.buchsbaum@opensource.tttech-industrial.com>
+Section: admin
+Build-Depends:
+   debhelper (>= 12),
+   dh-python,
+   python3,
+   python3-setuptools,
+   git,
+   libssl-dev,
+   libpciaccess-dev,
+   libsystemd-dev,
+   libevent-dev,
+   libxml2-dev,
+   libxml2-utils,
+   libusb-1.0-0-dev,
+   libblkid-dev,
+   e2fslibs-dev,
+   pkg-config,
+   libnuma-dev,
+   libsdl2-dev,
+   libpixman-1-dev,
+   liblz4-tool,
+   linux-libc-dev (>= 4.20),
+   libdrm-dev,
+   libcjson-dev,
+   flex,
+   bison,
+   xsltproc,
+   clang-format,
+   bc,
+   acpica-tools (>= 20200326),
+   python3-lxml,
+   python3-defusedxml,
+   python3-xmlschema (= 1.9.2-1),
+   python3-elementpath (= 2.4.0-1)
```

Comment:
(>= 2.4.0-1) Debian maintainer is on jump straight to https://salsa.debian.org/debian/elementpath/-/tree/debian/2.5.0-1
 no 2.4.0-1 on debian repos